libr/core/cmd_quit.c

```diff
@@ -19,27 +18,36 @@ static int cmd_quit(void *data, const char *input) {
 		r_core_cmd_help (core, help_msg);
 		break;
 	case '!':
-		if (input[1] == '!')
+		if (input[1] == '!') {
 			r_config_set (core->config, "scr.histsave", "false");
+		}
 		core->num->value = -1;
 		return -2;
 	case '\0':
 		core->num->value = 0LL;
 		return -2;
 	default:
-		if (*input == ' ')
+		if (*input == ' ') {
 			input++;
-		if (*input)
+		}
+		if (*input) {
 			r_num_math (core->num, input);
-		else core->num->value = 0LL;
-		if (*input == 'y')
-			core->num->value=5;
-		else if (*input == 'n')
-			core->num->value=1;
-		if (input[1] == 'y')
-			core->num->value+=10;	
-		else if (input[1] == 'n')
-			core->num->value+=2;	
+		}
+		else {
```

Comment:
After the while->chop change looks good to merge

> On 10 Jul 2016, at 21:38, Maijin <notifications@github.com> wrote:
> 
> In libr/core/cmd_quit.c:
> 
> >  			r_num_math (core->num, input);
> > -		else core->num->value = 0LL;
> > -		if (*input == 'y')
> > -			core->num->value=5;
> > -		else if (*input == 'n')
> > -			core->num->value=1;
> > -		if (input[1] == 'y')
> > -			core->num->value+=10;	
> > -		else if (input[1] == 'n')
> > -			core->num->value+=2;	
> > +		}
> > +		else {
> } else see other file for examples. also see ./sys/indent.sh for indenting file
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>